### ISSUE TITLE
Definition of is-qc-open

### ISSUE THREAD

**Issue by felixwellen** [felixwellen/synthetic-geometry#35]
I think our definition of open propositions is wrong. ```is-qc-open``` says that a proposition ```P``` is open iff there are $n\in\mathbb N$ and a vector ```v``` of length n such that *there is an ```i```* such that $P$ is equivalent to ```v i``` being invertible. But we should ask ```P``` to be equivalent to the existence of an ```i``` such that ```v i``` is invertible.
This means that the proof that open propositions are double negation stable does not work as it is now. Don't know if this affects anything else.

**Comment by MatthiasHu** [comment]
It looks right to me:
https://github.com/felixwellen/synthetic-geometry/blob/927e3414e3e520d0b52fbc9e1bff8429cd98c3cb/SyntheticGeometry/Open.lagda.md?plain=1#L48

**Comment by felixwellen** [comment]
Ah. I misread... I thought ```std-``` means standard in the sense of standard open....
Should we rename that?

**Comment by MatthiasHu** [comment]
Right, the name `std-qc-open-prop` is not very good. But I don't know what to call it right now.

**Comment by MatthiasHu** [comment]
By the way the D(f) should I think be called "basic open" to be clear (not "standard open").

**Comment by MatthiasHu** [comment]
Maybe call `std-qc-open-prop` just `contains-invertible-prop`!

**Comment by felixwellen** [comment]
Yes. I guess then it is all good and it's just me ;-)
Standard open in the sense we have now actually fits well with standard finite.

**Comment by felixwellen** [comment]
> Maybe call `std-qc-open-prop` just `contains-invertible-prop`!

That would have prevented me from misreading.

**Comment by felixwellen** [comment]
Not sure about the ```-prop```, but I think it is a good name.

**Comment by MatthiasHu** [comment]
> Not sure about the `-prop`

Yes, I see. An alternative would be to bundle the hProp right away and just have:
```agda
contains-invertible : {n : ℕ} → FinVec ⟨ k ⟩ n → hProp _
contains-invertible v = ∥ Σ[ i ∈ Fin _ ] (v i) ∈ k ˣ ∥ₚ
```

**Comment by felixwellen** [comment]
Yes, I think this (with ```∃```) should be the best way to do it (unless we use the non-prop version a lot, but I don't think we do or will).
